lms/static/js/instructor_dashboard/membership.js

```diff
@@ -38,10 +42,8 @@ such that the value can be defined later than this assignment (file load order).
                 add_btn_label: 'Add Member',
                 add_handler: function() {}
             });
-            templateHtml = $('#member-list-widget-template').html();
-            edx.HtmlUtils.setHtml(
-                this.$container, window.Mustache.render(templateHtml, edx.HtmlUtils.HTML(memberListParams))
-            );
+            templateHtml = window.Mustache.render($('#member-list-widget-template').html(), memberListParams);
+            edx.HtmlUtils.setHtml(this.$container, edx.HtmlUtils.HTML(templateHtml));
```

Comment:
Nit: ``setHtml`` safely escapes its input parameter if it is a string, so you don't need to also call ``HtmlUtils.HTML``.